src/textual/widgets/_progress_bar.py

```diff
@@ -72,27 +96,47 @@ def __init__(
         self._start_time = None
         self._percentage = None
 
+    def validate_colorscheme(self, colorscheme: str) -> str:
+        if colorscheme is None:
+            colorscheme = "default"
+        if colorscheme not in _VALID_BAR_COLORSCHEMES:
+            raise InvalidBarColorScheme(
+                f"Valid bar color shemes are {friendly_list(_VALID_BAR_COLORSCHEMES)}"
+            )
+        return colorscheme
+
+    def validate_thickness(self, thickness: int) -> int:
+        if thickness is None:
+            thickness = 1
+        if thickness not in range(3):
+            raise InvalidBarThickness(f"Valid thickness are {range(3)}")
+        return thickness
+
     def watch__percentage(self, percentage: float | None) -> None:
         """Manage the timer that enables the indeterminate bar animation."""
         if percentage is not None:
             self.auto_refresh = None
         else:
             self.auto_refresh = 1 / 15
 
+    def watch_colorscheme(self, colorscheme: str):
+        self.colorscheme = colorscheme
+
+    def watch_thickness(self, thickness: str):
+        self.thickness = thickness
```

Comment:
It is _very_ likely that you _don't_ need these two methods.

Reply:
Fixed.